acceptance/chaos_test.go

```diff
@@ -19,35 +19,43 @@
 package acceptance
 
 import (
+	"bytes"
+	"database/sql"
+	"flag"
 	"fmt"
-	"net/rpc"
+	"math/rand"
 	"strings"
 	"sync"
 	"sync/atomic"
 	"testing"
 	"time"
 
-	"github.com/cockroachdb/cockroach/client"
-	"github.com/cockroachdb/cockroach/roachpb"
 	"github.com/cockroachdb/cockroach/testutils"
 	"github.com/cockroachdb/cockroach/util/log"
 	"github.com/cockroachdb/cockroach/util/randutil"
 	"github.com/cockroachdb/cockroach/util/stop"
 )
 
-// TestChaos starts up a cluster and, for each node, a worker writing to
-// independent keys, while nodes are being killed and restarted continuously.
-// The test measures not write performance, but cluster recovery.
+var maxTransfer = flag.Int("max-transfer", 999, "Maximum amount to transfer in one transaction.")
+var numAccounts = flag.Int("num-accounts", 999, "Number of accounts.")
+
+// TestChaos starts up a cluster with an "accounts" table.
+// It starts transferring money between accounts, while nodes are
+// being killed and restarted continuously.
+// The test doesn't measure write performance, but cluster recovery.
+// TODO(vivek): Expand this test to check that write performance
+// is unaffected by chaos.
 func TestChaos(t *testing.T) {
 	c := StartCluster(t)
 	defer c.AssertAndStop(t)
 
 	num := c.NumNodes()
+	if num <= 0 {
+		log.Fatal("%d nodes in cluster", num)
```

Comment:
`vet` doesn't care about packages, it only looks at [function names](https://github.com/golang/tools/blob/a6e3cc80e87d58d7fdb23da870845b456969b929/cmd/vet/print.go#L60-L81), so it should complain about this. Maybe it doesn't look at files that are excluded by a build tag.

However, `Info`/`Infof` isn't on its list, so it doesn't cover most of our use of the `log` package. If we can, we should add `-printfuncs` to our `vet` invocation (although last time I looked at this there were some complexities related to the difference between `go vet` and `go tool vet`. Maybe we've fixed that with the use of `-shadow` now.